src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -54,6 +59,16 @@
 
   private static final String DEFAULT_SNIPPET_FILE = "discovery_fragment.snip";
 
+  private static final Map<String, Class<? extends SampleMethodToViewTransformer>>
+      SAMPLE_METHOD_TO_VIEW_TRANSFORMER_MAP =
+          ImmutableMap.of(
+              JAVA,
```

Comment:
nit: if we can keep keys and values in the same line, it will be easier for humans to parse